app/models/data_csv.rb

```diff
@@ -453,15 +429,37 @@ def self.update_with_accreditation
     query_str += 'AND accreditations.accreditation_status IS NOT NULL '
     query_str += "AND LOWER(accreditations.csv_accreditation_type) = 'institutional'; "
 
-    query_str += 'UPDATE data_csvs SET '
-    query_str += 'caution_flag_reason = CONCAT(data_csvs.caution_flag_reason,'
-    query_str += "'Accreditation (', accreditations.accreditation_status, '), ')"
-    query_str += ' FROM accreditations '
-    query_str += 'WHERE data_csvs.cross = accreditations.cross '
-    query_str += 'AND accreditations.cross IS NOT NULL '
-    query_str += %(AND LOWER(accreditations.periods) LIKE '%current%' )
-    query_str += 'AND accreditations.accreditation_status IS NOT NULL '
-    query_str += "AND LOWER(accreditations.csv_accreditation_type) = 'institutional'; "
+    query_str += 'UPDATE data_csvs SET caution_flag_reason = '
```

Comment:
@mphprogrammer Can you leave a comment in a follow-up PR that explains what the SQL here is doing. It's well formatted and clear from a code standpoint, but it's hard for someone following in your footprints to know the [business] purpose of the change and what it's trying to accomplish should someone ever need to maintain it.